cpp/source/deal.II/geometry.h

```diff
@@ -44,35 +44,64 @@ class Geometry
 
   virtual ~Geometry() = default;
 
-  inline std::shared_ptr<dealii::distributed::Triangulation<dim> const>
+  dealii::types::boundary_id get_anode_boundary_id() const
+  {
+    return _anode_boundary_id;
+  }
+
+  dealii::types::boundary_id get_cathode_boundary_id() const
+  {
+    return _cathode_boundary_id;
+  }
```

Comment:
I see :-( The problem is that right now we know theses quantities by reading them from the database.